src/GitLink/Extensions/PdbExtensions.cs

```diff
@@ -15,32 +15,21 @@ namespace GitLink
 
     public static class PdbExtensions
     {
-        public static Dictionary<string, string> VerifyPdbFiles(this PdbFile pdbFile, IEnumerable<string> files)
+        public static IEnumerable<string> FindMissingOrChangedSourceFiles(this PdbFile pdbFile)
         {
             Argument.IsNotNull(() => pdbFile);
 
-            var missing = new Dictionary<string, string>(StringComparer.OrdinalIgnoreCase);
-            var actualFileChecksums = (from x in files
-                                       select new KeyValuePair<string, string>(Hex.Encode(Crypto.GetMd5HashForFiles(new[] { x }).First().Item1), x)).ToDictionary(x => x.Value, x => x.Key);
-
             foreach (var checksumInfo in pdbFile.GetChecksums())
             {
-                var file = checksumInfo.Key;
-                var checksum = checksumInfo.Value;
+                string file = checksumInfo.Key;
+                string expectedChecksum = checksumInfo.Value;
+                string actualChecksum = Hex.Encode(Crypto.GetMd5HashForFile(file));
 
-                if (!actualFileChecksums.ContainsValue(checksum))
+                if (expectedChecksum != actualChecksum)
```

Comment:
This is already fixed in a later iteration, where I do a buffer compare of the hash, skipping the hex encoding altogether for better performance.